tests/v2/test_0355-mixins.py

```diff
@@ -54,59 +54,13 @@ def weighted_add(self, other):
         ],
         with_name="Point",
     )
-    two = ak._v2.Array(
-        [
-            [{"x": 0.9, "y": 1}, {"x": 2, "y": 2.2}, {"x": 2.9, "y": 3}],
-            [],
-            [{"x": 3.9, "y": 4}, {"x": 5, "y": 5.5}],
-        ],
-        with_name="Point",
-    )
     wone = ak._v2.Array(
         ak._v2.operations.with_field(one, abs(one), "weight"),
         with_name="WeightedPoint",
     )
-    wtwo = ak._v2.Array(
-        ak._v2.operations.with_field(two, abs(two), "weight"),
-        with_name="WeightedPoint",
-    )
 
     assert to_list(one + wone) == [
         [{"x": 2, "y": 2.2}, {"x": 4, "y": 4.4}, {"x": 6, "y": 6.6}],
         [],
         [{"x": 8, "y": 8.8}, {"x": 10, "y": 11.0}],
     ]
-    assert to_list(wone + wtwo) == [
-        [
-            {
-                "x": 0.9524937500390619,
-                "y": 1.052493750039062,
-                "weight": 2.831969279439222,
-            },
-            {"x": 2.0, "y": 2.2, "weight": 5.946427498927402},
-            {
-                "x": 2.9516640394605282,
-                "y": 3.1549921183815837,
-                "weight": 8.632349833200564,
-            },
-        ],
-        [],
-        [
-            {
-                "x": 3.9515600270076154,
-                "y": 4.206240108030463,
-                "weight": 11.533018588312771,
-            },
-            {"x": 5.0, "y": 5.5, "weight": 14.866068747318506},
-        ],
-    ]
-    assert to_list(abs(one)) == [
-        [1.4866068747318506, 2.973213749463701, 4.459820624195552],
-        [],
-        [5.946427498927402, 7.433034373659253],
-    ]
-    assert to_list(one.distance(wtwo)) == [
-        [0.14142135623730953, 0.0, 0.31622776601683783],
-        [],
-        [0.4123105625617664, 0.0],
-    ]
```

Comment:
Did we get rid of this because it's flukey?

Reply:
Yes. It's been failing in the last digits on only some platforms. This is a test of the not-completely-done v2 inside `ak._v2.*`, so I don't think we need to work very hard to save it.

Reply:
In principle, we could remove all of `ak._v2` and `tests/v2` from `main-v1`, but I don't want to make such a large diff in this PR.

Reply:
I'd prefer to keep some of the tests so we don't fully regress, but I agree this is transitional code and should be treated as such. We didn't make long-term guarantees about v2-in-v1.